hal/armv7a/imx6ull/imx6ull.c

```diff
@@ -417,4 +417,7 @@ void _hal_platformInit(void)
 
 	/* remain in run mode in low power */
 	*(imx6ull_common.ccm + ccm_clpcr) &= ~0x3;
+
+	/* kick watchdog power down counter */
+	*(imx6ull_common.wdog + wdog_wmcr) &= 0;
```

Comment:
why `&=` instead of `=`? Do we need to read out the value?

Reply:
There is no need to read out the value, changed to `=`.